api/GPUDevice.json

```diff
@@ -1436,6 +1436,49 @@
             "deprecated": false
           }
         },
+        "color_space_display-p3": {
+          "__compat": {
+            "description": "<code>display-p3</code> color space",
+            "tags": [
+              "web-features:webgpu"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "121",
+                "notes": "Currently supported on ChromeOS, macOS, and Windows only."
+              },
+              "chrome_android": {
+                "version_added": "121"
```

Comment:
Nit: You can mirror since it's the same version.

Reply:
The only reason I didn't mirror is that I thought it would also mirror the note, and "Currently supported on ChromeOS, macOS, and Windows only." is not relevant to Chrome Android?

I'm pretty sure I am correct here, as the BCD linter didn't tell me to mirror it when I ran the tests.

Reply:
Gotcha!